test/Hooks.t.sol

```diff
@@ -182,7 +182,8 @@ contract HooksTest is Test, Deployers, GasSnapshot {
                 beforeSwap: false,
                 afterSwap: false,
                 beforeDonate: false,
-                afterDonate: false
+                afterDonate: false,
+                noOp: false
```

Comment:
Add `assertFalse(Hooks.shouldAllowNoOp(hookAddr));` All the other tests where you add noOp: false needs this statement as well